gatsby-config.js

```diff
@@ -14,7 +14,8 @@ module.exports = {
       options: {
         alias: {
           react: path.resolve(__dirname, 'node_modules', 'react')
-        }
+        },
+        defaultBranch: 'main',
```

Comment:
Thanks for filing this fix, @kyontan!

@siddharthkp (👋🏻  FR) this change looks correct to me as per https://primer.style/doctocat/usage/customization#default-branch, but the "edit this page" links in the [preview deploy for this PR](https://presentations-git-fork-kyontan-patch-2-primer.vercel.app/) still appear to point to the `master` branch. Any idea what's up?

Reply:
The option is supplied to the wrong plugin :)

Reply:
Ah, It's my mistake 🙏  I fixed.